src/internal_events/syslog.rs

```diff
@@ -28,7 +28,7 @@ impl InternalEvent for SyslogUdpReadError {
     }
 
     fn emit_metrics(&self) {
-        counter!("udp_read_errors", 1, "mode" => "udp");
+        counter!("read_errors_total", 1, "mode" => "udp");
```

Comment:
Don't shoot me, but we should probably call this `socket_read_errors_total` since we have other `socket_*` metrics (ex: `socket_errors`).